src/demo.js

```diff
@@ -45,20 +31,16 @@ function Annotation(props) {
         annotation: { message, line_offset, column_offset, head, tail }
     } = props;
 
-    return (
-        <div className="output__item annotation">
-            <div className="output__key">
-                <code className="annotation__name">Error on line {line_offset + 1}</code>
-            </div>
-            <div className="output__value">
-                <pre className="annotation__slice">{head}</pre>
-                <pre className="annotation__label">
-                    {indent(column_offset)}⮤ {message}
-                </pre>
-                <pre className="annotation__slice">{tail}</pre>
-            </div>
-        </div>
-    );
+    return [
+      <dt>Error on line {line_offset + 1}</dt>,
+      <dd>
+        <pre className="annotation__slice">{head}</pre>
+        <pre className="annotation__label">
+        {indent(column_offset)} {message}
```

Comment:
Please re-add `⮤` after the indent.

Reply:
I have to say I don't understand what it's doing, but readded.

Reply:
It points to the exact location of the error:

![screenshot from 2017-10-06 09-14-12](https://user-images.githubusercontent.com/265818/31267112-334ed1ce-aa77-11e7-8e44-8a082ccd13d8.png)

Reply:
not on my machine: https://i.imgur.com/eA1XCVy.png